parsers/CO.py

```diff
@@ -91,19 +93,22 @@ def fetch_historical_consumption(
 ) -> list[dict[str, Any]]:
     demand_list = TotalConsumptionList(logger)
     # Convert datetime to local time
-    target_arrow_in_tz = arrow.get(target_datetime).to(TZ)
+    target_datetime_in_tz = datetime.fromisoformat(target_datetime).replace(tzinfo=TZ)
```

Comment:
You can not pass a datetime object to fromisoformat, you can manipulate it directly.

I also think this should use `.astimezone()` instead of replace, but double check this.